### PR TITLE
Update

### PR DESCRIPTION
**Pull request by prateek-senapati**
Hey, Aman! I have made some changes in various parts of the repository. Hope you will like them :) If you find them apt, I would love to see you add them to your repository.

### CODE REVIEW

**Comment by prateek-senapati** [comment]
If you decide to add these changes to your repository, you may also want to edit the `README.md` files in both the `C++` repository and the `DSA-Implementation` repository.

**Comment by prateek-senapati** [comment]
The reason for creating a new directory, viz. `sorting-algos`, is that some of the C++ implementations present in the directory `Sorting` are not printing expected outputs, such as time duration. You may want to test them.

File: C++/Sorting/HeapSort.cpp [hunk 1]
```diff
@@ -0,0 +1,59 @@
+// C++ implementation of Heap Sort algorithm
```
Comment (AmanRaj1608) [inline C++/Sorting/HeapSort.cpp]: Looks good

File: C++/Readme.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
-# Data Structure Implementation in C++
+# Data Structures Implementation in C++
 
-**Implementation of Data Structure in my favourite language.**
+**Implementation of various Data Structures in my favorite language: C++.**
 
-Contributor - 
-    <li> Array List [sid-tiw](https://github.com/sid-tiw)
\ No newline at end of file
+Contributor(s):
```
Comment (AmanRaj1608) [inline C++/Readme.md:5]: You can add your name here

**Review by AmanRaj1608 — APPROVED** [review]

**Comment by AmanRaj1608** [comment]
> The reason for creating a new directory, viz. `sorting-algos`, is that some of the C++ implementations present in the directory `Sorting` are not printing expected outputs, such as time duration. You may want to test them.

@prateek-senapati
Thanks for `sorting-algo`.
Can you tell the wrong output codes name by creating an issue in repo.
Yes I will add relevant changes

**Pull request merged**